libs/judicial-system/types/src/lib/dates.ts

```diff
@@ -0,0 +1,27 @@
+import addDays from 'date-fns/addDays'
+
+const VERDICT_APPEAL_WINDOW_DAYS = 28
+const FINE_APPEAL_WINDOW_DAYS = 3
+
+const APPEAL_WINDOW_DAYS = 3
+
+const setEndOfDay = (date: Date) => new Date(date.setHours(23, 59, 59, 999))
+
+export const hasDatePassed = (deadline: Date) => Date.now() > deadline.getTime()
+
+export const getIndictmentAppealDeadlineDate = ({
+  baseDate,
+  isFine,
+}: {
+  baseDate: Date
+  isFine?: boolean
+}) => {
+  const windowDays = isFine
+    ? FINE_APPEAL_WINDOW_DAYS
+    : VERDICT_APPEAL_WINDOW_DAYS
+  const deadlineDate = addDays(baseDate, windowDays)
+  return setEndOfDay(deadlineDate)
+}
+
+export const getAppealDeadlineDate = ({ baseDate }: { baseDate: Date }) =>
```

Comment:
Please consider rewriting the parameters as such

`export const getAppealDeadlineDate = (baseDate: Date) =>`

So that you can call this function like so 

`getAppealDeadlineDate(baseDate)`

instead of 

`getAppealDeadlineDate({baseDate})`